packages/client/components/ActivityLibrary/ActivityDetails/ActivityDetails.tsx

```diff
@@ -41,7 +41,9 @@ graphql`
 export const query = graphql`
   query ActivityDetailsQuery {
     viewer {
-      tier
+      featureFlags {
+        templateLimit
```

Comment:
-1 we've removed the template limits feature flag in a recent PR https://github.com/ParabolInc/parabol/pull/8359, so this can be removed now